apps/web/pagesAndRewritePaths.js

```diff
@@ -1,15 +1,17 @@
 const glob = require("glob");
 const { getSubdomainRegExp } = require("./getSubdomainRegExp");
 /** Needed to rewrite public booking page, gets all static pages but [user] */
-let pages = (exports.pages = glob
-  .sync("pages/**/[^_]*.{tsx,js,ts}", { cwd: __dirname })
+// Pages found here are excluded from redirects in beforeFiles in next.config.js
+const BEFORE_REWRITE_EXCLUDE_PAGES = (exports.pages = glob
+  .sync("{pages,app}/**/[^_]*.{tsx,js,ts}", { cwd: __dirname })
```

Comment:
- Because `/pages` directory no longer had `reschedule` and `booking`, these were not excluded from the redirects